preparing dataloader
preparing dataloader
epoch progress DISC:   0%|          | 0/10 [00:00<?, ?it/s]
iterations progress DISC:   0%|          | 0/1250 [00:00<?, ?it/s][Aoutput1:  tensor([[ 0.0303,  0.0365,  0.0138,  ...,  0.0379,  0.0142, -0.0782],
        [ 0.0121, -0.0282, -0.0044,  ...,  0.0026,  0.0156,  0.0043],
        [-0.0011, -0.0250, -0.0146,  ..., -0.0020,  0.0153,  0.0131],
        ...,
        [ 0.0309,  0.0054,  0.0079,  ...,  0.0029,  0.0137, -0.0403],
        [-0.0261, -0.0140, -0.0154,  ...,  0.0185, -0.0159, -0.0267],
        [ 0.0346,  0.0205, -0.0030,  ...,  0.0111,  0.0185, -0.0382]],
       device='cuda:0', grad_fn=<CatBackward>)
output2:  tensor([[-0.0006,  0.0399, -0.0588,  ..., -0.0047,  0.0784, -0.0068],
        [ 0.0096, -0.0250, -0.0054,  ...,  0.0036,  0.0174,  0.0012],
        [-0.0110, -0.0498, -0.0112,  ...,  0.0053,  0.0086,  0.0150],
        ...,
        [ 0.0092, -0.0087, -0.0293,  ..., -0.0293,  0.0413,  0.0229],
        [ 0.0305,  0.0803,  0.0202,  ...,  0.0333,  0.0178, -0.0933],
        [-0.0054,  0.0361, -0.0466,  ...,  0.0247,  0.0332, -0.0214]],
       device='cuda:0', grad_fn=<CatBackward>)
iterations progress DISC:   0%|          | 0/1250 [00:00<?, ?it/s]
epoch progress DISC:   0%|          | 0/10 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train_disc.py", line 135, in <module>
    main()
  File "train_disc.py", line 100, in main
    output3 = disc(output1, output2)
  File "/root/miniconda3/envs/hi/lib/python3.8/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
TypeError: forward() takes 2 positional arguments but 3 were given
